README.md

```diff
@@ -4,8 +4,10 @@ An implementation of the browser DOM—primarily for SSR—in Deno. Implemented
 Rust, WASM, and obviously, Deno/TypeScript.
 
 ## Example
+
+*mod.ts*
 ```typescript
-import { DOMParser, Element } from "https://deno.land/x/deno_dom@v0.1.2-alpha3/deno-dom-wasm.ts";
+import { DOMParser, Element } from "https://deno.land/x/deno_dom@v0.1.2-alpha4/deno-dom-native.ts";
```

Comment:
Please import from `https://deno.land/x/deno_dom/deno-dom-wasm.ts`. I primarily advertise the WASM backend because it's cross-platform and needs no permissions to run. The native backend is primarily for its performance advantages, but because of all the Deno permissions it needs, it shouldn't be the preferred backend.

Reply:
And yeah, that should also be
```js
import { DOMParser,  } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
import type { Element } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
```
to be prepared for Deno 1.5.0

Reply:
You're absolutely right. I've almost finished fixing all the nits, however to get examples working properly a **tsconfig.json** with `importsNotUsedAsValues` set to `remove` is required.